### ISSUE TITLE
Training nano-plus-m with backbone shufflenetv2_1.0x from scratch get significantly lower APs

### ISSUE THREAD

**Issue by NrealWJX** [RangiLyu/nanodet#402]
Hi, I train nano-plus-m (image size is 384*384) with default settings and get 0.262 AP at 300 epoch and 0.251 AP at 90 epoch.

However, when I train from scratch (set pretrain=False in yaml), I get 0.213 AP at 90 epoch.

Have you try to train those models from scratch? How are those results?

- with pretrained weights
![image](https://user-images.githubusercontent.com/96856636/153789479-cef01051-ae72-4a85-8176-77a1a0c0b788.png)

- without pretrained weights
![image](https://user-images.githubusercontent.com/96856636/153789527-20fa0ef8-344e-452d-8f80-2c1382dd2ae1.png)

**Comment by RangiLyu** [comment]
For tiny backbones, pre-training is very important.

**Comment by NrealWJX** [comment]
> For tiny backbones, pre-training is very important.

@RangiLyu  Thx for your reply. I will pre-train the backbone before using it on target datasets.

I also read the tips from yolov5: [Tips-for-Best-Training-Results](https://github.com/ultralytics/yolov5/wiki/Tips-for-Best-Training-Results). However it recommends to pre-train or not according to dataset scale instead of model size. How do you like this?

<img width="927" alt="image" src="https://user-images.githubusercontent.com/96856636/154243590-e609b729-42d3-49c4-b233-7d246813a613.png">
